student.rb

```diff
@@ -1,11 +1,16 @@
 require_relative 'person'
```

Comment:
- [x] I think it would be nice if you create the Person `has-many`  Rentals association between class `Person` and `Rental` as required in the project requirements. You can achieve this by adding a `@rentals` array and an `add_rental` method to the `Person` class, the same way you did in class `Book`. Please see the example below.

## Example code
```ruby
require_relative 'nameable'

class Person < Nameable
  attr_accessor :age, :name
  attr_reader :id

  def initialize(age, name = 'unknown', parent_permission: true)
    super()
    @id = Random.rand(1..1000)
    @name = name
    @age = age
    @parent_permission = parent_permission
    @rentals = []
  end

  def of_age?
    @age >= 18
  end

  def can_use_services?
    @age >= 18 || @parent_permission
  end

  def correct_name
    @name
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end
end
``` 

## Project requirements
![Screenshot from 2023-06-24 12-45-02](https://github.com/Metaverse-Nabeel/OOP-school-library/assets/16392046/ddc032a3-4aca-48aa-8f6c-471f351ef123)